docs/components.rst

```diff
@@ -7,15 +7,25 @@ REANA system is composed of multiple separated components that are developed
 independently. The components are usually published as Docker images. The
 components collaborate via REST APIs.
 
-reana-server
+reana-client
 ------------
 
-REANA component for managing analyses.
+REANA command line client for end users.
 
-- source code: `<https://github.com/reanahub/reana-server>`_
-- releases: `<https://hub.docker.com/r/reanahub/reana-server/tags>`_
-- known issues: `<https://github.com/reanahub/reana-server/issues>`_
-- documentation: `<https://reana-server.readthedocs.io/>`_
+- source code: `<https://github.com/reanahub/reana-client>`_
+- releases: `<https://hub.docker.com/r/reanahub/reana-client/tags>`_
+- known issues: `<https://github.com/reanahub/reana-client/issues>`_
+- documentation: `<https://reana-client.readthedocs.io/>`_
+
+reana-cluster
+-------------
+
+REANA component providing utilities to manage cluster instance.
+
+- source code: `<https://github.com/reanahub/reana-cluster>`_
+- releases: `<https://hub.docker.com/r/reanahub/reana-cluster/tags>`_
```

Comment:
should we link here to Github releases instead of DockerHub since it is CLI? (it would also apply for `reana-client`)